### ISSUE TITLE
Support for Luarocks

### ISSUE THREAD

**Issue by mayocream** [dependabot/dependabot-core#5211]
I think supporting Luarocks will be beneficial for all Lua developers and OpenResty developers.
I didn't see any previous discussion about this one, so I opened it.

**Comment by Ismoh** [comment]
I fully agree! I am going to implement a workaround using GitHub Action Workflow for this. When done, I'll share it!

**Comment by mayocream** [comment]
I am working on an implementation to support LuaRocks :) 
- #6301

**Comment by Ismoh** [comment]
Nice, then I won't investigate time!

**Comment by Ismoh** [comment]
@mayocream any news/progress on this?

**Comment by mayocream** [comment]
I restarted the work on implementing this feature, PR opened: #13711 :)
